src/Types/RedeemerTag.purs

```diff
@@ -26,3 +36,23 @@ derive instance Ord RedeemerTag
 
 instance Show RedeemerTag where
   show = genericShow
+
+instance Enum RedeemerTag where
+  pred = genericPred
+  succ = genericSucc
+
+instance Bounded RedeemerTag where
+  top = Reward
+  bottom = Spend
+
+instance BoundedEnum RedeemerTag where
```

Comment:
I don't see how can this instance be used. I think the order of the constructors is rather arbitrary. Please provide a justification or remove (together with `Bounded` and `Enum`).

Reply:
I didn't write this code so I don't really know why it was added. It's part of this commit: [b5d6c32e2](https://github.com/Plutonomicon/cardano-transaction-lib/commit/b5d6c32e2f5b7c6eed1002908aacfc84654d1acc#diff-4393a67a738272abd3fe0cadecc576ddf83d873a34fa56638c79c9e65eb97676)